lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -144,6 +144,9 @@ private function build($data, $display = 'teaser') {
         $this->timing = $timing;
 
         $this->services = $this->getServices();
+
+        $mobile_app = $this->getMobileAppDate();
+        $this->mobile_app = $mobile_app;
```

Comment:
You can probably consolidate this down to one thing since you're not doing anything else w/ the `$mobile_app` variable :)